portality/templates/application_form/assed_application.html

```diff
@@ -10,32 +10,27 @@
 
 {% block editor_content scoped %}
 <body id="apply">
-    <main class="container">
+    <main class="container page-content">
+        <h1>{{ obj.bibjson().title }}</h1>
```

Comment:
Due to the chance of some odd characters in titles it might be worth using the `| safe` filter so they're not escaped by jinja (you should be able find some examples elsewhere)

Reply:
NP! Will have to change it everywhere -- this was already being used in other places, I think.

Reply:
Also, what kind of odd characters? Is there sometimes markup in journal titles?

Reply:
or maybe escaping it is correct - let's keep it how you've done it

Reply:
No problem! I had started collecting different "edge cases" for journal titles in non-Latin scripts. Perhaps we could collect any other weird cases too where there is potentially  "bad" data? 

https://docs.google.com/document/d/1koBSLp4Zj9ktLeOqwX5o5q9YaJeKubOQXZleK2UDyQU/edit#heading=h.yp6yhhe31rv8